pyanaconda/ui/gui/spokes/lib/resize.py

```diff
@@ -419,7 +419,7 @@ def _update_action_buttons(self):
             self._delete_button.set_sensitive(False)
 
     def _update_reclaim_button(self, got):
-        self._resize_button.set_sensitive(got + self._initial_free_space >= self._required_size)
```

Comment:
It looks like you can completely remove `self._initial_free_space`, because this is the only place where we use its value.

Reply:
Right, PR updated.